### DISCUSSION TITLE
Secureboot for BlackArch

### DISCUSSION THREAD

**Discussion by ghost** [BlackArch/blackarch#4326] [Ideas]
Since Solus 4.4 secure boot is now supported. When you first boot the ISO, and, if you have secure boot enabled in your UEFI firmware; you will have to perform the one-time-step of manually enrolling the Solus certificate. The following guide will walk you through this. If you already have Solus installed and wish to enable secure boot, skip ahead [here](https://help.getsol.us/docs/user/quick-start/installation/secure-boot/#enabling-secure-boot-on-an-existing-install).

Note that this only applies to machines with UEFI firmware, if your machine uses the older BIOS firmware you can safely ignore this article. If you wish to avoid having to do this step then you may disable secure boot in your machine's UEFI firmware interface.

https://help.getsol.us/docs/user/quick-start/installation/secure-boot/

I think maybe if arch linux based ones use something like this, they would be able to also find a fix for that whole secure boot thing, I know fedora, kubuntu, ubuntu and OpenSuse already have their own method, but I think the registering an .cer file into the secure boot that then allows you to boot into linux with secure boot on, seems to work on all my pc's so far, for the Solus version, I wanted to post this somewhere like here, anyway I hope this can be put into other linux distros too, it might help people that don't want to touch their bios settings.

**Comment by D3vil0p3r** [comment]
@noraj can we move this to https://github.com/BlackArch/blackarch/discussions and close this issue?
